src/main/java/seedu/address/logic/parser/util/InternshipUtil.java

```diff
@@ -0,0 +1,38 @@
+package seedu.address.logic.parser.util;
+
+import static seedu.address.model.util.ItemUtil.COMPANY_NAME;
+import static seedu.address.model.util.ItemUtil.INTERNSHIP_NAME;
+
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.company.CompanyItem;
+import seedu.address.model.internship.InternshipItem;
+
+/**
+ * Gets the internship from a company, checking the indexes validity while doing so.
+ */
```

Comment:
shouldn't this be the javadoc for the method below?
isn't util just a general purpose utility class?

Reply:
Edited the name of method and javadocs for increased clarity, do help to check before merging!